### PR TITLE
Add a getFeedURL API for the autoUpdater

### LINKED ISSUE #6454: Add getFeedURL API to autoUpdater
* Electron version: 1.2.6
* Operating system: osx yosemite

Do the autoUpdater objects on osx and windows have different properties? On windows, when i do ```console.log(JSON.stringify(autoUpdater));```
I see the properties `domain`, `_events`, `_eventsCount`, and `updateURL`. However, doing the same thing on osx only allows me to see the properties `_events` and `_eventsCount`. 

I want to access `updateURL` in my code but I'm wondering whether that property is accessible. Maybe I should submit a PR for a `getFeedURL` method?

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Implements the request `getFeedURL` API.

Closes #6454

### CODE REVIEW

File: lib/browser/api/auto-updater/auto-updater-win.js [hunk 1]
```diff
@@ -19,6 +19,10 @@ AutoUpdater.prototype.quitAndInstall = function () {
   return app.quit()
 }
 
+AutoUpdater.prototype.GetFeedURL = function () {
```
Comment (kevinsawicki) [inline lib/browser/api/auto-updater/auto-updater-win.js]: I think should be `getFeedURL`, lower case `G`

File: atom/browser/api/atom_api_auto_updater.cc [hunk 2]
```diff
@@ -109,6 +109,7 @@ void AutoUpdater::BuildPrototype(
     v8::Isolate* isolate, v8::Local<v8::ObjectTemplate> prototype) {
   mate::ObjectTemplateBuilder(isolate, prototype)
       .SetMethod("checkForUpdates", &auto_updater::AutoUpdater::CheckForUpdates)
+      .SetMethod("getFeedURL", &auto_updater::AutoUpdater::GetFeedURL)
```
Comment (kevinsawicki) [inline atom/browser/api/atom_api_auto_updater.cc]: Perhaps we should add an assert for this in `spec/api-auto-updater-spec.js`

File: lib/browser/api/auto-updater/auto-updater-win.js (hunk 1 above)
Comment (MarshallOfSound) [inline lib/browser/api/auto-updater/auto-updater-win.js]: Woops

File: atom/browser/api/atom_api_auto_updater.cc (hunk 2 above)
Comment (MarshallOfSound) [inline atom/browser/api/atom_api_auto_updater.cc]: @kevinsawicki The autoUpdater on macOS fails to `setFeedURL` when the application is unsigned.  The test would only work for Windows I believe

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**